src/conponents/comment/Comment.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export function Comment({ name, body, email }) {
+  const emailLink = `mailto:${email}`;
+
+  return (
+    <div className="list-group mt-3">
+      <figure className="list-group-item">
```

Comment:
Not sure if `figure` tag is suitable here semantically.
https://www.w3schools.com/tags/tag_figure.asp